docs/general/ad-filtering/create-own-filters.md

```diff
@@ -581,8 +581,8 @@ Rules with the `$to` modifier are supported by AdGuard for Windows, Mac, and And
 * [`$subdocument`](#subdocument-modifier)
 * [`$websocket`](#websocket-modifier)
 * [`$xmlhttprequest`](#xmlhttprequest-modifier)
-* [`$object-subrequest` (deprecated)](#object-subrequest-modifier)
-* [`$webrtc` (deprecated)](#webrtc-modifier)
+* [`$object-subrequest` (removed)](#object-subrequest-modifier)
```

Comment:
do we need to keep them?

Reply:
we may warn users by [AGLint](https://github.com/AdguardTeam/AGLint/) about it